packages/dx-react-grid/docs/guides/export.md

```diff
@@ -68,7 +68,7 @@ The following demo shows export in action:
 
 ## Supported Grid Features
 
-Exported data can be transformed in the Excel document as it is in the Grid. For this, pass Grid settings to the following `GridExporter` properties:
+Data shaping settings applied to the Grid can be also applied to the exported Excel document. For this, pass Grid settings to the following `GridExporter` properties:
```

Comment:
Can we say:
To apply a grid's data shaping settings to an exported Excel document, pass the settings to the following `GridExporter` properties:

Reply:
```suggestion
To apply the Grid's data shaping settings to an exported Excel document, pass the settings to the following `GridExporter` properties:
```